registration-processor/pre-processor/registration-processor-packet-uploader-stage/src/main/java/io/mosip/registration/processor/packet/uploader/service/impl/PacketUploaderServiceImpl.java

```diff
@@ -403,8 +402,7 @@ private boolean scanFile(final byte[] input, String id, final Map<String, InputS
 			description.setMessage(PlatformErrorMessages.RPR_PUM_PACKET_VIRUS_SCANNER_SERVICE_FAILED.getMessage());
 			description.setCode(PlatformErrorMessages.RPR_PUM_PACKET_VIRUS_SCANNER_SERVICE_FAILED.getCode());
 			dto.setStatusCode(RegistrationExceptionTypeCode.VIRUS_SCANNER_SERVICE_FAILED.toString());
-			dto.setStatusComment(trimExpMessage.trimExceptionMessage(
-					StatusUtil.VIRUS_SCANNER_SERVICE_NOT_ACCESSIBLE.getMessage() + e.getMessage()));
+			dto.setStatusComment(RegistrationExceptionTypeCode.VIRUS_SCANNER_SERVICE_FAILED.toString());
```

Comment:
I have marked this as not a issue and discussed with Basawaraj, So this change is not reuqired, but include a space between StatusUtil.VIRUS_SCANNER_SERVICE_NOT_ACCESSIBLE.getMessage() and e.getMessage()

Don't include this bug id in in the PR title

Reply:
reverted this change and included space.